src/main/kotlin/no/skatteetaten/aurora/mokey/model/AuroraStatus.kt

```diff
@@ -1,6 +1,133 @@
 package no.skatteetaten.aurora.mokey.model
 
-data class AuroraStatus(val level: AuroraStatusLevel, val comment: String, val statuses: List<HealthStatusDetail>)
+import no.skatteetaten.aurora.utils.value
+import java.time.Duration
+import java.time.Instant
+
+class AuroraStatus {
+
+    val AVERAGE_RESTART_OBSERVE_THRESHOLD = 20
+    val AVERAGE_RESTART_ERROR_THRESHOLD = 100
+    val DIFFERENT_DEPLOYMENT_HOUR_THRESHOLD = 2
+
+    val level: AuroraStatusLevel
+    val comment: String
+    val statuses: List<HealthStatusDetail>
+
+    constructor(level: AuroraStatusLevel, comment: String, statuses: List<HealthStatusDetail>) {
```

Comment:
Hvorfor trenger vi denne konstuktøren egentlig? Er det noen ganger at den er i bruk?

Reply:
For tester, for å enkelt kunne lage en instans av AuroraStatus

Reply:
Hva om dette bare er en data classe og så har vi en statisk metode for å beregene?

Reply:
Det er også en mulighet, og jeg liker nok det bedre enn å ha det som en service. Men jeg syns fortsatt vi burde vurdere å bruke objektorientering der det gir mening. Og jeg syns dette er et av tilfellene, hvor det gir mening å blande state med behavior.